app/routes/static.rb

```diff
@@ -36,25 +36,27 @@ class Static < Core
         erb :"site/how_it_works_newbie"
       end
 
-      get '/cli' do
-        # Add OS detection logic & redirect here
-        erb :"site/cli", locals: { docs: X::Docs::CLI.new, page: :overview }
-      end
-
-      get '/cli/mac' do
-        erb :"site/cli", locals: { docs: X::Docs::CLI.new, page: :mac }
-      end
-
-      get '/cli/windows' do
-        erb :"site/cli", locals: { docs: X::Docs::CLI.new, page: :windows }
-      end
-
-      get '/cli/linux' do
-        erb :"site/cli", locals: { docs: X::Docs::CLI.new, page: :linux }
-      end
-
-      get '/cli/install' do
-        erb :"site/cli", locals: { docs: X::Docs::CLI.new, page: :install }
+      namespace '/clients' do
```

Comment:
Does `namespace '/clients'` do the exact same thing as typing `get '/clients/cli'`?